playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -151,22 +151,22 @@
   tags:
     - run
 
-- name: Create .config/openstack directory
+- name: Ensure /etc/openstack directory exists
+  become: yes
   file:
-    path: /root/.config/openstack
+    path: /etc/openstack
     state: directory
     owner: root
     group: root
     mode: '0755'
-  tags:
-    - run
 
 - name: Save default clouds.yaml file
+  become: yes
   template:
-    src: clouds.yaml.j2
-    dest: /root/.config/openstack/clouds.yaml
     owner: root
     mode: '0640'
```

Comment:
With these permission other users wont be able to access that file. OTOH if that if clouds.yml contains credentials we don't want to make those `0644`.